index.html

```diff
@@ -247,11 +247,15 @@ <h2>Top Security</h2>
                         online daily to run your daily
                         transactions our mobile 
                         app is ready for download at no cost
-                    </p>
```

Comment:
Kindly close your HTML element. eg <p> always needs </p>

Reply:
- [ ] Please always include an explanation, a "why", for each requested change. For example:

This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*

This request lacks a "why": *Why don't you use an article tag here instead of a div?*

- [Optional] Consider using a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.